packages/data/src/settings/reducer.js

```diff
@@ -55,7 +64,8 @@ const receiveSettings = (
 						data: state[ group ] ? [ ...state[ group ].data, ...groupIds ] : groupIds,
 						error,
 						lastReceived: time,
-						persisting,
+						persisting: state[ group ] ? state[ group ].persisting : false,
```

Comment:
Good guards here but this will potentially add `undefined` to the `persisting` property instead of a boolean. So instead, what about doing:

```
persisting: state [ group ] ? Boolean( state[ group ].persisting ) : false,
```